frontend/pages/journey/new.vue

```diff
@@ -250,20 +282,64 @@ function copyToClipboard() {
                     </form>
                 </fieldset>
             </div>
-            <div class="z-10">
+            <div class="mt-2 flex items-center justify-center px-4 font-nunito">
                 <div
-                    class="relative flex flex-row items-end justify-between border-b border-natural-200"
+                    id="template-section"
+                    class="w-full rounded-xl border-2 border-natural-300 bg-natural-50 dark:border-natural-800 dark:bg-natural-900 sm:w-2/4 md:w-2/5"
                 >
-                    <SvgPeopleBackpackMap class="hidden h-full lg:flex" />
+                    <h3
+                        class="-m-0.5 ml-4 mt-0.5 text-lg font-semibold text-natural-800 dark:text-natural-200"
+                    >
+                        <T key-name="template.suggestions" />
+                    </h3>
+                    <div v-if="status === 'pending'" class="mt-1 flex w-full">
+                        <ProgressSpinner class="w-10" />
+                    </div>
                     <div
-                        class="mt-2 flex h-full w-full flex-row items-end justify-between sm:mt-0 lg:absolute lg:inset-0 lg:justify-end"
+                        v-else-if="suggestions.length < 1"
+                        class="mt-1 w-full text-center italic"
                     >
-                        <SvgWomanSuitcaseLeft />
-                        <SvgWomanSuitcaseRight class="ml-10 mr-5" />
+                        <T key-name="template.suggestions.none" />
+                    </div>
+                    <div
+                        v-else
+                        id="template-suggestions"
+                        class="mt-1 flex w-full flex-col"
+                    >
+                        <TemplateSuggestion
```

Comment:
wenn ich nach Vienna z.b Suche hätte ich gerne Vinna und Vienna, Austria Templates

Reply:
This may be due to template destination name no longer being sent to the backend? @EinSev

Reply:
yes, my mistake. should be fixed with the next push. the retrieved full address was set but also resetted again before the request was made